src/yamlfix/model.py

```diff
@@ -1,17 +1,24 @@
 """Define program entities like configuration value entities."""
 
 from typing import Optional
+from enum import Enum
 
 from maison.schema import ConfigSchema
 
 
+class StyleSequence(Enum):
+    flow_style = 1
+    block_style = 2
+    as_is = 3
+
+
 class YamlfixConfig(ConfigSchema):
     """Configuration entity for yamlfix."""
 
     allow_duplicate_keys: bool = False
     config_path: Optional[str] = None
     explicit_start: bool = True
-    flow_style_sequence: Optional[bool] = True
+    style_sequence: Optional[StyleSequence] = StyleSequence.flow_style
```

Comment:
Optional would no longer be required, when this gets converted to an enum:

```suggestion
    flow_style_sequence: YamlNodeStyle = YamlNodeStyle.flow_style
```

Also - I would keep the name, as it implies that this config is for sequences and not for mappings.

Reply:
Also, when renaming these options in `YamlfixConfig`, accordingly the documentation needs to be updated as well -> https://github.com/lyz-code/yamlfix/blob/main/docs/index.md?plain=1#L204 (e.g.: name of the option, name of the generated env var, available options)